### PR TITLE
Add: IBC Messages

### LINKED ISSUE #192: feature: ibc messages
Currently all ibc related messages are `unknown`. Research and update all commonly used ibc messages so that we can display them in a ui friendly format

### PR DESCRIPTION
**Pull request by MonikaCat**
## Description

Fixes #192 

## Checklist
- [x] Ran Linting
- [x] Targeted PR against correct branch.
- [x] Linked to Github issue with discussion and accepted design OR link to spec that describes this work.
- [ ] Wrote unit tests.
- [ ] Added an entry to the `CHANGELOG.md` file.
- [x] Re-reviewed `Files changed` in the Github PR explorer.

### CODE REVIEW

File: src/models/msg/transfer/msg_transfer.ts [hunk 1]
```diff
@@ -0,0 +1,25 @@
...
+    static fromJson(json: any) {
+      return new MsgTransfer({
+        json,
+        type: json['@type'],
+        signer: json.signer,
+      });
+    }
+}
+
+export default MsgTransfer;
```
Comment (ryuash) [inline src/models/msg/transfer/msg_transfer.ts:25]: I haven't checked the other messages but this specific tx msg `1BCD30C1740E7949D68B9383C0865CD04656599B2045B75A5B27CEA34FA627C0` crashed the dev server. It displays correctly if i display it as unknown.

File: src/models/msg/types.ts [hunk 2]
```diff
@@ -1 +1 @@
-export type Categories = 'bank' | 'crisis' | 'distribution' | 'governance' | 'slashing' | 'staking' | 'profiles' | 'others';
+export type Categories = 'bank' | 'crisis' | 'distribution' | 'governance' | 'slashing' | 'staking' | 'profiles' | 'client' | 'channel' | 'connection' | 'transfer' | 'others';
```
Comment (ryuash) [inline src/models/msg/types.ts:1]:
```
export type Categories = 'bank' | 'crisis' | 'distribution' | 'governance' | 'slashing' | 'staking' | 'profiles' | 'ibc' | 'ibc-transfer' | 'others'
```

File: src/styles/theme/index.ts [hunk 3]
```diff
@@ -114,6 +114,8 @@ export const common = {
         six: '#497BFF',
         seven: '#FF7549',
         eight: '#EB3AA4',
+        nine: '1E3B93',
+        ten: '4F12D1',
```
Comment (ryuash) [inline src/styles/theme/index.ts:118]: I'm surprised these colors are showing up? You updated it here but didn't update the Tag component. 

Nothing to fix here though. I'm going to merge a PR with 10 new colors today.

**Review by ryuash — CHANGES_REQUESTED** [review]
Please Fix

File: src/models/index.ts [hunk 4]
```diff
@@ -37,6 +37,32 @@ import MsgDtagRefuseTransfer from './msg/profiles/msg_dtag_refuse_transfer';
...
+import MsgTimeout from './msg/channel/msg_timeout';
+import MsgTimeoutOnClose from './msg/channel/msg_timeout_on_close';
+import MsgConnectionEnd from './msg/connection/msg_connection_end';
+import MsgConnectionOpenAck from './msg/connection/msg_connection_open_ack';
+import MsgConnectionOpenConfirm from './msg/connection/msg_connection_open_confirm';
+import MsgConnectionOpenInit from './msg/connection/msg_connection_open_init';
+import MsgConnectionOpenTry from './msg/connection/msg_connection_open_try';
+import MsgCounterpartyConnection from './msg/connection/msg_counterparty';
+import MsgVersion from './msg/connection/msg_version';
+import MsgTransfer from './msg/transfer/msg_transfer';
```
[the remark this answers is not in the document]
Comment (MonikaCat) [reply src/models/index.ts]: Ok I updated the code as requested.

File: src/models/msg/types.ts (hunk 2 above)
Comment (MonikaCat) [reply src/models/msg/types.ts]: The message categories have been updated

File: src/models/msg/transfer/msg_transfer.ts (hunk 1 above)
Comment (MonikaCat) [reply src/models/msg/transfer/msg_transfer.ts]: I see, I updated MsgTransfer types and tested it. It displays the information correctly now.

File: src/styles/theme/index.ts (hunk 3 above)
Comment (MonikaCat) [reply src/styles/theme/index.ts]: I have rebased this branch with base branch and applied your changes

**Review by ryuash — APPROVED** [review]

**Review by ryuash — APPROVED** [review]

**Pull request merged**